src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/ClaimSettledDashboardNotificationHandler.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.claimant;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.client.DashboardApiClient;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.DashboardNotificationsParamsMapper;
+import uk.gov.hmcts.reform.dashboard.data.ScenarioRequestParams;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackParams.Params.BEARER_TOKEN;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.CREATE_DASHBOARD_NOTIFICATION_FOR_CLAIM_SETTLED_FOR_CLAIMANT1;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.DashboardScenarios.SCENARIO_AAA7_CLAIMANT_INTENT_CLAIM_SETTLED_EVENT_CLAIMANT;
+
+@Service
+@RequiredArgsConstructor
+public class ClaimSettledDashboardNotificationHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = List.of(CREATE_DASHBOARD_NOTIFICATION_FOR_CLAIM_SETTLED_FOR_CLAIMANT1);
+    public static final String TASK_ID = "CreateClaimSettledDashboardNotificationsForClaimant1";
+    private final DashboardApiClient dashboardApiClient;
+    private final DashboardNotificationsParamsMapper mapper;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
+            callbackKey(ABOUT_TO_SUBMIT), this::configureScenarioForClaimSubmission
+        );
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse configureScenarioForClaimSubmission(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        String authToken = callbackParams.getParams().get(BEARER_TOKEN).toString();
+
+        dashboardApiClient.recordScenario(caseData.getCcdCaseReference().toString(),
```

Comment:
Do you need to check if is?
Full admit + Pay immediately
Part admit + Pay immediately

Reply:
Yes... Initially, I added that but 3rd Scenario in the ticket says so I kept it generic

**Scenario 3: Claimant uses the 'settle' event**

Given the claimant uses the settle event at any stage in the claim journey
When the claimant displays their dashboard
Then all notifications for this case are deleted, as per DELETE (list) - as above
And a notification should be displayed to advise the claimant that their claim is now settled
And the <date> should be dynamically updated in the notice 
And tasks in the task list should have status (as per Tasklist - above)